src/common/Configuration/Config.cpp

```diff
@@ -259,8 +269,8 @@ std::string ConfigMgr::GetValueDefault<std::string>(std::string const& name, std
     {
         if (showLogs)
         {
-            LOG_ERROR("server.loading", "> Config: Missing name %s in config, add \"%s = %s\"",
-                name.c_str(), name.c_str(), def.c_str());
+            FMT_LOG_ERROR("server.loading", "> Config: Missing name {} in config, add \"{} = {}\"",
+                name, name, def);
         }
```

Comment:
This one must be removed, we **DO NOT** want to be forced having all the confs of the `.dist` file inside our `.conf`

Reply:
This message will be displayed if all (`.conf` and `.conf.dist`) configs do not have this option (all core and all modules files)

Reply:
ah! so the message is a bit confusing, let me suggest a small  change there